docker/tensorflow-aarch64/cpu_info.sh

```diff
@@ -104,6 +110,26 @@ function set_target {
       eigen_l2_cache="1024*1024"
       eigen_l3_cache=
     ;;
+    neoverse-v1 )
+      cpu="neoverse-v1"
+      tune="neoverse-v1"
+      arch="armv8.4-a"
+      blas_cpu="NEOVERSEV1"
+      blas_ncores=64
+      eigen_l1_cache="64*1024"
+      eigen_l2_cache="1024*1024"
+      eigen_l3_cache=
+    ;;
+    neoverse-n2 )
+      cpu="neoverse-n2"
+      tune="neoverse-n2"
+      arch="armv8.5-a"
```

Comment:
I believe the N2 will be an Armv9-a core (https://developer.arm.com/ip-products/processors/neoverse/neoverse-n2). 
I think it would be worth adding a comment here that this will need to be updated when the build has a compiler with appropriate support.

Reply:
Agree, leaving a note would be more clear to avoid confusion.